src/app/root/auth/login/login.component.ts

```diff
@@ -1,8 +1,68 @@
-import { Component } from '@angular/core';
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { Router } from '@angular/router';
+import { Subject, takeUntil } from 'rxjs';
+import { InputTypes } from 'src/app/types/enums/input-types.enum';
+import { hashPassword } from '../passwordHash';
+import { samePasswords } from '../passwords.validator';
 
 @Component({
   selector: 'app-login',
   templateUrl: './login.component.html',
   styleUrls: ['./login.component.scss']
 })
-export class LoginComponent {}
+export class LoginComponent implements OnInit, OnDestroy{
+  
+  private unsubscribe$ = new Subject();
```

Comment:
This can be removed with ngOnDestroy method, cause no need to use it in LogIn component